make_md.py

```diff
@@ -49,7 +54,14 @@ def convert_from_bib(myline):
         print(myentry)
         print("We are in trouble ! ")
     if "eprint" in myentry_dict:
-        return "["+myentry_dict["title"]+"](https://arxiv.org/abs/"+myentry_dict["eprint"]+")"
+        paper=""
+        if "doi" in myentry_dict:
+            paper=", published [{0}](https://doi.org/{0})".format( myentry_dict["doi"] )
```

Comment:
@vlimant Looking [at the results on your fork's branch](https://github.com/vlimant/HEPML-LivingReview/tree/cms_llp_tagger) the "published ..." looks a bit busy and cluttered. What do you think about just having the format of

> [Deep learning in color: towards automated quark/gluon](https://arxiv.org/abs/1612.01551) [DOI: [10.1007/JHEP01(2017)110](https://doi.org/10.1007/JHEP01(2017)110)]

 ? Also, this is Python 3 so you can use f-strings and make everything much nicer.

Reply:
How about simply

[Name](arXiv) [Published](doi link)

?

This will take up fewer lines in the README.

Reply:
I like [DOI:...] @matthewfeickert 
aff0166 for [Name](arXiv) [Published](doi link) looks weird

Reply:
Agreed. If the goal is to keep things short I would go with either

> [DOI]

or 

> [Published]

with the text inside the square-brackets hyperlinked to the DOI.

Reply:
with [DOI] it looks nice imo

Reply:
I agree!